builders/interoperability/xcm/xcm-sdk/v1/reference.md

```diff
@@ -0,0 +1,315 @@
+    |  `ecosystem`  |          *Ecosystem*           |     Identifies the ecosystem the chain belongs to: `polkadot`, `kusama`, or `alphanet-relay`     |
+    | `isTestChain` |           *boolean*            |                                  Whether the chain is a TestNet                                  |
+    |     `key`     |            *string*            |                                        Identifies a chain                                        |
+    |    `name`     |            *string*            |                                      The name of the chain                                       |
+    |    `type`     |          *ChainType*           |                      The type of the chain: `parachain` or `evm-parachain`                       |
+    | `assetsData`  | *Map<string, ChainAssetsData>* |                           A list of the assets that the chain supports                           |
+    | `genesisHash` |            *string*            |                                  The hash of the genesis block                                   |
+    | `parachainId` |            *number*            |                                     The ID of the parachain                                      |
+    | `ss58Format`  |            *number*            | The [ss58 format](https://polkadot.js.org/docs/keyring/start/ss58/){target=_blank} for the chain |
+    |     `ws`      |            *string*            |                               The WebSocket endpoint for the chain                               |
+    |     `id`      |            *number*            |                            **For EVM parachains only** - The chain ID                            |
+    |     `rpc`     |            *string*            |                **For EVM parachains only** - The HTTP RPC endpoint for the chain                 |
+
+- `ChainAssetsData` - defines information needed to target the asset on the chain
+
+    |       Name       |      Type      |                       Description                       |
+    |:----------------:|:--------------:|:-------------------------------------------------------:|
+    |     `asset`      |    *Asset*     |            The asset's key and origin symbol            |
+    |   `balanceId`    | *ChainAssetId* |  The balance ID of the asset. Defaults to the asset ID  |
+    |    `decimals`    |    *number*    |          The number of decimals the asset has           |
+    |       `id`       | *ChainAssetId* |                      The asset ID                       |
+    |   `metadataId`   | *ChainAssetId* |              The metadata ID of the asset               |
+    |     `minId`      | *ChainAssetId* |               The minimum ID of the asset               |
+    | `palletInstance` |    *number*    | The number of the pallet instance the asset belongs to  |
+    |      `min`       |    *number*    | The minimum amount of the asset that can be transferred |
```

Comment:
@ekenigs this section in particular might need some help, haha  do you have better descriptions for some of these items?